libvast/src/system/configuration.cpp

```diff
@@ -116,6 +130,8 @@ caf::error configuration::parse(int argc, char** argv) {
     for (const auto& [old, new_] : replacements)
       if (option == old)
         option = new_;
+  // FIXME(MV): why does this need to be here? The comment doesn't explain the
+  // need for this. Is there a hidden side-effect?
```

Comment:
If you figure this out before merging I'm fine with this PR overall.

Reply:
I clarified the comment.